cli/README.md

```diff
@@ -165,6 +166,7 @@ path|status|description
 [jobs/automl-standalone-jobs/cli-automl-classification-task-bankmarketing/cli-automl-classification-task-bankmarketing.yml](jobs/automl-standalone-jobs/cli-automl-classification-task-bankmarketing/cli-automl-classification-task-bankmarketing.yml)|[![jobs/automl-standalone-jobs/cli-automl-classification-task-bankmarketing/cli-automl-classification-task-bankmarketing](https://github.com/Azure/azureml-examples/workflows/cli-jobs-automl-standalone-jobs-cli-automl-classification-task-bankmarketing-cli-automl-classification-task-bankmarketing/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-automl-standalone-jobs-cli-automl-classification-task-bankmarketing-cli-automl-classification-task-bankmarketing.yml)|A Classification job using bank marketing
 [jobs/automl-standalone-jobs/cli-automl-forecasting-task-energy-demand/cli-automl-forecasting-task-energy-demand.yml](jobs/automl-standalone-jobs/cli-automl-forecasting-task-energy-demand/cli-automl-forecasting-task-energy-demand.yml)|[![jobs/automl-standalone-jobs/cli-automl-forecasting-task-energy-demand/cli-automl-forecasting-task-energy-demand](https://github.com/Azure/azureml-examples/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-energy-demand-cli-automl-forecasting-task-energy-demand/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-energy-demand-cli-automl-forecasting-task-energy-demand.yml)|A Time-Series Forecasting job using energy demand dataset
+[jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml](jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml)|[![jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau](https://github.com/Azure/azureml-examples/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-github-dau-cli-automl-forecasting-task-github-dau/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-github-dau-cli-automl-forecasting-task-github-dau.yml)|A Time-Series Forecasting job using Github DAU dataset that trains only the TCNForecaster model.
```

Comment:
It seems, we have the same entry two times.

Reply:
Thanks! I have updated it. It seems the readme.py generated duplicate values. Some duplicate values are also present in the main branch- [here](https://github.com/Azure/azureml-examples/blob/main/cli/README.md). Example- "jobs/automl-standalone-jobs/cli-automl-forecasting-bike-share/cli-automl-forecasting-task-bike-share.yml"